packages/app-builder/src/components/Decisions/DecisionsList.tsx

```diff
@@ -26,9 +29,28 @@ type Column =
   | 'score'
   | 'outcome';
 
+interface DecisionVM {
```

Comment:
what does VM stand for ?

Reply:
ok I think I got it (view model?) but since it's showing up in just 4 places, let's maybe keep it full text ?